### ISSUE TITLE
AWS::Lambda::Function - ZipFile

### ISSUE THREAD

**Issue by joshuayhwu** [aws-cloudformation/cloudformation-coverage-roadmap#883]
## 2. Scope of request 
Support for 
## 6. Category (required) - Will help with tagging and be easier to find by other users to +1

1. Compute: **Lambda**

## 7. Any additional context (optional)

Attempting to follow the demo [here](https://aws.amazon.com/blogs/machine-learning/classify-a-large-number-of-images-with-amazon-rekognition-and-aws-batch/), which is a demo for classifying images with batch and amazon rekognition created 3 years ago. 

I manually updated the parameter for [this yaml file](https://s3.amazonaws.com/aws-bigdata-blog/artifacts/RekognitionBatch/cfn/network-elasticsearch.yaml) as elasticbeanstalk supports 64bit Amazon Linux 2 v5.4.1 running Node.js 12  [as stated here ](https://docs.aws.amazon.com/elasticbeanstalk/latest/platforms/platforms-supported.html#platforms-supported.nodejs) (tried 10.x and 14.x as well, but same error). 

Then I create the stack with the [second yaml file](https://s3.amazonaws.com/aws-bigdata-blog/artifacts/RekognitionBatch/cfn/master.yaml)  that links to a [lambda function yaml file](https://s3-cloudbeginners-bucket-jw.s3.amazonaws.com/lambda-functions+-+updated.yaml) using nodejs12.22.1 at runtime for lambda functions. I then got [this error](https://s3-cloudbeginners-bucket-jw.s3.amazonaws.com/OAR_failure3.PNG). This shouldn't be a problem according to issue on #80, thus I created this separate issue. Very grateful for any help!

**Comment by micahduron** [comment]
Hello, all nodejs runtimes now support the `ZipFile` attribute, so you should be unblocked.
